src/content/packages-and-plugins/swift-package-manager/index.md

```diff
@@ -0,0 +1,6 @@
+---
+layout: toc
+title: Swift Package Manager
```

Comment:
maybe: "Swift Package Manager for Flutter"

Reply:
Updated!